app/src/main/java/com/infomaniak/drive/ui/LaunchActivity.kt

```diff
@@ -184,7 +188,11 @@ class LaunchActivity : AppCompatActivity() {
     }
 
     private fun handleShortcuts() {
-        intent.extras?.getString(SHORTCUTS_TAG)?.let { extrasMainActivity = MainActivityArgs(shortcutId = it).toBundle() }
+        intent.extras?.getString(SHORTCUTS_TAG)?.let {
+            extrasMainActivity = MainActivityArgs(shortcutId = it).toBundle()
+            if (it == Utils.Shortcuts.FEEDBACK.id)
```

Comment:
I think you could rename extrasMainActivity to mainActivityExtras and create a loginActivityExtras to be coherent instead of creating this 'isHelpShortcutPressed" Boolean